src/index.html

```diff
@@ -4,9 +4,78 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="../styles.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="month">
+      <div class="month__day-1  month--start-Tuesday">
+      </div>
+      <div class="month__day-2">
```

Comment:
i don't think you should hardcode the cell numbers , maybe you can use `for loop` & `nth child`